lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -190,6 +190,17 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
     Value rhs = convertScalarToDtype(b, loc, payloadArgs[1], dtype);
     return b.create<arith::AndIOp>(loc, lhs, rhs);
   }
+  if (auto logicalOr = dyn_cast<AtenLogicalOrOp>(op)) {
+
+    MLIRContext * context = op->getContext();
```

Comment:
Make sure to run `git clang-format` on your changes. Some of the spacing in this file is not right

Reply:
ack

Reply:
The formatting still does not look right. In your case, because you have 6 commits, you should do `git clang-format HEAD~6`. If you then do `git status` you can see the modified files. (another option is to squash the commits, and then do `git clang-format HEAD~1`)